legal-api/src/legal_api/services/filings/validations/admin_freeze.py

```diff
@@ -0,0 +1,42 @@
+# Copyright © 2022 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Validation for the Admin Freeze filing."""
+from http import HTTPStatus
+from typing import Dict, Optional
+
+from flask_babel import _ as babel  # noqa: N813, I004, I001; importing camelcase '_' as a name
+# noqa: I004
+from legal_api.errors import Error
+from legal_api.models import Business
+from ...utils import get_str  # noqa: I003; needed as the linter gets confused from the babel override above.
+
+
+def validate(business: Business, put_back_on: Dict) -> Optional[Error]:
```

Comment:
references to `put_back_on` in this function should be `admin_freeze` right?

Reply:
correct, made the correction.